homework/list.js

```diff
@@ -1,44 +1,36 @@
 'use strict';
 
-(function() {
+(function(exports) {
+var TodoListManager = function(){
+	this._listNoteContent = [];
+	this._wrapper = null;	
+};
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
-
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+TodoListManager.prototype = {
+	
+  onNoteOpen(event) {
```

Comment:
It's a convention to prefix method with `on` for the corresponding function, so it should be renamed as 'onNoteClick'.